src/plugins/controls/timer/component/countdown.js

```diff
@@ -272,10 +282,21 @@ export default function countdownFactory($container, config) {
             this.render($container);
         })
         .on('render', function() {
+            var timeString = timeEncoder.encode(this.remainingTime / precision);
+            var timeArr;
+            var hours;
+            var minutes;
             $time = $('.time', this.getElement());
+            $timeAriaLabel = $('#timerbox-aria-label', this.getElement());
 
             if (this.config.showBeforeStart === true) {
-                $time.text(timeEncoder.encode(this.remainingTime / precision));
+                if (timeString) {
+                    timeArr = timeString.split(':');
+                    hours = timeArr[0];
+                    minutes = timeArr[1];
+                    $timeAriaLabel.text( __('timer: %s hours %s minutes to answer', hours, minutes ));
```

Comment:
Since the same code is presented twice, please consider factoring it in a single place.